### PR TITLE
navbar & hero section

### PR DESCRIPTION
**Pull request by ayoub-bani**
still some minor changes to be done

### CODE REVIEW

**Comment by rabraghib** [comment]
### Good Job 👌!!
just a little small detail. 
It will be better to verticly align the title next to the logo in the middle.
wich is as ez as adding one line of css.
```css
.logo--color {
      ...
      vertical-align: middle;
}
```

> as long as `logo--color` class is only used in the logo title

**+ Its mobile friendly.**
EDIT: I am talking about the navbar. the hero section needs more work.

**Comment by ayoub-bani** [comment]
> ### Good Job 👌!!
> just a little small detail.
> It will be better to verticly align the title next to the logo in the middle.
> wich is as ez as adding one line of css.
> 
> ```css
> .logo--color {
>       ...
>       vertical-align: middle;
> }
> ```
> 
> > as long as `logo--color` class is only used in the logo title
> 
> **+ Its mobile friendly.**
> EDIT: I am talking about the navbar. the hero section needs more work.

ok thanks Raghib for your suggestion i will use it 
but maybe we will redo the project because they will use Tailwind CSS instead of bootstrap

File: src/app/_shared/navbar/navbar.component.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+.navbar-brand {
+    width: 40px;
+    font-family: 'futura medium', sans-serif;
+}
+.active--color {
+    color: #DB5B69 !important;
```
Comment (airaamane) [inline src/app/_shared/navbar/navbar.component.css:7]: I suggest using CSS global variables for colours, will be easier in the future to change the theme or add dark mode for example :)

File: src/app/home/video-banner/video-banner.component.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    margin-block-start: 5rem;
+}
+  /* Fonts classes */
+.futura-light-font {
+    font-family: 'futura light', sans-serif;
+    width: 50rem;
+}
+
+.adjust {
+    height: 50rem;
```
Comment (airaamane) [inline src/app/home/video-banner/video-banner.component.css:23]: I would suggest using either rem or em or px throughout the app, mixing them leads to minor styling issues going forward

File: src/app/home/video-banner/video-banner.component.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    top: -4rem;
+    right: -10rem;
+}
+
+.custom-size {
+    font-family: 'futura light';
```
Comment (airaamane) [inline src/app/home/video-banner/video-banner.component.css:29]: same comment for fonts, I would keep these global, will be easier to change whenever the need arises

File: src/app/home/video-banner/video-banner.component.html [hunk 4]
```diff
@@ -1 +1,18 @@
-<p>video-banner works!</p>
+<section id="hero">
+    <div class="container">
+        <div class="row">
+        <div class="col-md-6">
+            <!-- <div class="d-flex space-top">
+                <span class="mr-2"><img class="shape" src="./assets/images/shape.png" alt=""></span>
+                <p class="custom-color">Last Episode</p>
+            </div> -->
+            <!-- <h1 class="futura-medium-font">NgMorocco</h1> -->
```
Comment (airaamane) [inline src/app/home/video-banner/video-banner.component.html:9]: is the comment needed? if not I'd suggest removing them!

**Review by airaamane — COMMENTED** [review]
I left some (minor) comments for your attention, hope you don't mind!

**Comment by ayoub-bani** [comment]
> I left some (minor) comments for your attention, hope you don't mind!

no worries i'm more than happy to share my code here so anyone interested can review it like you did that's why we join communities like this :)

**Comment by chihab** [comment]
Thanks a lot @ayoub-bani for your efforts.

We're going to use Tailwind so... 😊

**Pull request closed without merging**